contributors/devel/sig-scalability/kubemark-setup-guide.md

```diff
@@ -0,0 +1,71 @@
+## Introduction
+This document serves to understand how to set up kubemark cluster given that a base cluster (to run hollow-node pods) and separate master instance(s) (to act as master(s) for the hollow nodes) are already present.
+
+## Precondition
+You need two k8s clusters(kubemark cluster and external cluster) to set up a final kubemark cluster.
```

Comment:
This sounds kind of weird that you 'need a kubemark cluster to set up kubemark cluster'. Can you change it to 'need kubemark master to set up kubemakr cluster'?

Reply:
Done